ceph/ceph/values.yaml

```diff
@@ -276,12 +274,26 @@ bootstrap:
     }
     ensure_pool volumes 8
 
-ceph_local_pool:
-  failure_domain: host
-  subtree: rack
-  pg_num: "128"
-  num_rep: "3"
-  min_size: "2"
+# Uncomment below to enable mgr modules
```

Comment:
it's odd that the enabled list is different from the config list (and it would honestly be better if there were just one setting, but that may be enough work to not be worth it).  But anyway, my question as a user reading this is "so are there any modules enabled by default?"  That is, does *any* mgr module need to appear in ceph_mgr_enabled_modules, or what happens if it's empty?  If there is a default, are there modules that then can't be disabled because there's only an "enabled" variable?